### ISSUE TITLE
ACT Framework Requirements document is too general

### ISSUE THREAD

**Issue by moekraft** [w3c/wcag-act#5]
As reflected in survey feedback by Andrew Kirkpatrick's feedback in [WCAG WG Survey](https://www.w3.org/2002/09/wbs/35422/ACT_DPUB_Review_Nov_2016/results) requesting more information about ATAG and UAAG in the requirements document. 

In the Framework introduction we state, "There are currently many products available which aid their users in testing for conformance of accessibility standards such as WCAG 2.0." Our intent was to ensure the framework was applicable to WCAG 2.0 and beyond.

We should state that this framework is specifically for Web Content Accessibility Guidelines, either with no version or state 2.0 and later. We can then provide an applicability section that discusses how this framework can be applied to other standards. 

I am concerned that since this will become a normative document that we want to be careful not to be too generic but to give specifics as for which standard we are defining requirements.

Discussed on weekly WCAG ACT call, 11/16/16.

**Comment by WilcoFiers** [comment]
I'm not in favor of tying ACT to WCAG. I see the problem of a somewhat open ended target, but I think this is an important part. I totally get how that makes proving that the project succeeded difficult. I'd much rather us find a solution to that. One way we could go about it is to be explicit about which documents we want to target to prove that we are successful in making this framework fit different accessibility requirements. Something like:

"To show that the ACT Framework is capable of targeting different accessibility requirements, the following accessibility requirements are used: WCAG 2.0, ATAG 2.0, ARIA Authoring Practices 1.0 and EPUB Accessibility 1.0."

**Comment by moekraft** [comment]
Hi Wilco, I think we need to make this perfectly clear and be careful not to tie it to WCAG 2.0 if that is our intent, e.g. WCAG 2.0 Failure Techniques. Alternately, if this is intended to be a normative document on how to test for WCAG 2.x, we can have a separate Applicability section that talks about how this can be applied to ATAG, EPUG, UAAG.
